core/network/protocols/block_announce_protocol.hpp

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_NETWORK_BLOCKANNOUNCEPROTOCOL
+#define KAGOME_NETWORK_BLOCKANNOUNCEPROTOCOL
+
+#include <memory>
+#include "network/protocol_base.hpp"
```

Comment:
It think it should be either on the very top or in bottom block.

Reply:
sure